pkg/cmd/authenticated_cli_command.go

```diff
@@ -197,7 +197,7 @@ func (c *AuthenticatedCLICommand) GetSchemaRegistryClient(cmd *cobra.Command) (*
 			if u.Scheme != "http" && u.Scheme != "https" {
 				u.Scheme = "https"
 			}
-			configuration.BasePath = u.String()
+			configuration.Servers = srsdk.ServerConfigurations{{URL: u.String()}} //unsure
```

Comment:
This seems correct to me, have you tested this?

Reply:
yep, manually tested sr commands